tasks/67_find-elements/find-elements.js

```diff
@@ -0,0 +1,5 @@
+export function findElement(arr, func) {
+  for (let i = 0; i < arr.length; i++) {
+    if (func(arr[i])) return arr[i];
```

Comment:
Please try to remain consistency and use { } for if/else statement.

```suggestion
    if (func(arr[i])) {
      return arr[i];
    }
```